projects/kit/components/input-password/input-password-options.ts

```diff
@@ -0,0 +1,29 @@
+import {InjectionToken} from '@angular/core';
+import {TuiHintMode} from '@taiga-ui/core/enums';
+import {TuiSizeL, TuiSizeS} from '@taiga-ui/core/types';
+
+export interface InputPasswordOptions {
+    readonly hintMode: Readonly<TuiHintMode | null>;
+    readonly icons: Readonly<{
+        hide: (size: TuiSizeS | TuiSizeL) => string;
```

Comment:
Please use `PolymorpheusContent<TuiContextWithImplicit<TuiSizeS | TuiSizeL>>`

Reply:
OK, I've changed it